docs/API.md

```diff
@@ -139,12 +139,12 @@ Basic data type: object
 DS for [Delegation Signer](https://tools.ietf.org/html/rfc4034) references DNSKEY-records in the sub-delegated zone.
 
 Properties:
-* `"digest"`: A string, required. Either 40 or 64 hexadecimal characters (case insensitive).
+* `"digest"`: A string, required. Either 40, 64 or 96 hexadecimal characters (case insensitive).
```

Comment:
I am not sure that we really should have these values set here or in the validating code. This will prevent someone from using Zonemaster with an experimental digest type with another lenght.

Reply:
Are experimental digest types supported by stock Zonemaster::Engine and Zonemaster::LDNS today? And if not, would it be difficult to add that feature?

Reply:
I realize that the GUI only accepts pre-defined digest types, so if the focus of the API is the GUI and public systems (which it is) we can keep it as-is. If we get other requests then we could change it.